core/objectaction/object.go

```diff
@@ -256,6 +269,69 @@ func (t T) Options() actionrouter.T {
 	return t.T
 }
 
+func rsHumanRender(rs []actionrouter.Result) string {
+	var (
+		rsTree *tree.Tree
+		rsNode *tree.Node
+	)
+	type treeProvider interface {
+		Tree() *tree.Tree
+	}
+	s := ""
+	manyResults := len(rs) > 1
+	for i, r := range rs {
+		switch {
+		case errors.Is(r.Error, object.ErrDisabled):
+			if manyResults {
```

Comment:
same default output regardless node selector value ?
This may help client to diagnose from output